App/src/views/HomeView.js

```diff
@@ -1,7 +1,15 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
 import React from "react";
+import { Redirect, navigate } from "@reach/router";
+import { useUser, useUserUpdater } from "../contexts/user";
```

Comment:
Why do you put 'useUserUpdater' in this line? if you don't use them in this file.